Platform/OpenCanopy/BitmapFont.c

```diff
@@ -693,6 +740,13 @@ GuiGetLabel (
     InitialWidthOffset = 0;
   }
 
+  // Apply palette because Apple does it with stored labels
```

Comment:
Please use surrounding `//` for upper and lower lines for comments.

Reply:
I wanted to discuss if we need this code or not. You say that Apple has pngs in addition to the label format. Are they already used or it's for the future?

Reply:
- They do not have PNG images ondisk at least presently.
- Disk labels can be in two formats, with the second being BGRA format, which support I recently added to disklabel utility.
- Apple uses PNG or BMP in the firmware for prebuilt labels (e.g. `EFI Boot`), but their protocol accepts any format, including `icns` and disk label (either type).
- Ondisk disk labels are processed with the same protocol, so it can technically process them in any format, but `icns` will be likely unsupported.
- Presently I do not know of any ondisk labels using anything but type 1 (alpha-only 8-bit format).

I am not sure we need to bother to support all other formats to be honest, maybe with the time of if we find them being used in the wild.

Reply:
The current question is: do we want to post-process our rendered labels (decrease their color depth by mapping them via clut and inverse clut), because that's what happens when generating and decoding .disklabel files, or just display what we rendered, supposing that Apple doesn't do it with builtin labels? Is "EFI Boot" saved after such processing or it's raw CoreText-rendered image?